uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -129,6 +145,31 @@ define([
       });
     });
 
+    describe('$scope.role', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      it('defaults to as admin', function () {
```

Comment:
all the initialization logic of the controller should be under the same section. You already have a `describe init()` block, put this there
```js
describe('init()', function () {
  beforeEach(function () {
    // ...
    initController();
  });

  it('sets the role as admin by default', function () {
  });

  it('correctly formats date time in document', function () {
  });
});